src/jit/lir.h

```diff
@@ -72,36 +73,25 @@ class LIR final
         unsigned ReplaceWithLclVar(Compiler* compiler, unsigned blockWeight, unsigned lclNum = BAD_VAR_NUM);
     };
 
-    //------------------------------------------------------------------------
-    // LIR::Range: Represents a contiguous range of LIR nodes. Provides a
-    //             variety of utilites that modify the LIR contained in the
-    //             range.
-    //
-    class Range
+    class ReadOnlyRange
     {
```

Comment:
Class header comment for this class? Either Range or ReadOnlyRange comment (or both) should specifically point out the other, and when to use one versus the other.